R/dependencies.R

```diff
@@ -391,6 +391,9 @@ renv_dependencies_find_dir <- function(path, root, depth) {
   # list children
   children <- renv_dependencies_find_dir_children(path, root, depth)
 
+  # notify about number of children
+  renv_condition_signal("renv.dependencies.count", list(path = path, count = length(children)))
```

Comment:
Is this used anywhere?

Reply:
Yes, see https://github.com/rstudio/renv/pull/1603/files#diff-5d849f3782ddfdd6b0053b841b4be729f4fdaa33c9262caf6757aa185e5f5658R927-R929, which fills in the `count` variable, and https://github.com/rstudio/renv/pull/1603/files#diff-5d849f3782ddfdd6b0053b841b4be729f4fdaa33c9262caf6757aa185e5f5658R946-R948 where we use it.